client/app/scripts/liveblog-security.service.js

```diff
@@ -32,7 +32,7 @@ angular.module('liveblog.security', [])
             return $rootScope.currentUser._id === archive.original_creator || canPublishAPost();
         }
         function canAccessSettings(archive) {
-            return canCreateABlog() && isUserOwnerOrAdmin(archive);
+            return canCreateABlog() && (isUserOwnerOrAdmin(archive) || isMemberOfBlog(archive));
```

Comment:
after the change you made here, all the members of the blogs will have the rights to edit the settings, not only the editors, right?